noize_client:
  hdf5_open_acc_service_name: hdf5_open_segmentation_service
  visualize_service_name: visualize_cloud_service
  hdf5_open_file_path: /home/ericlab/tsuchida/2023_01/annotation/Semseg/HV8/temp/try_20.hdf5

